docs/payments-refunds/create-payments.md

```diff
@@ -157,12 +157,12 @@ curl https://api.mobilepay.dk/v1/payments/{PAYMENT_ID}/cancel \
 ```
 
 Keep in mind, that payment can be cancelled in all states except `captured`. If you've already captured payment, refunding is your only option.
-If you cancel a reserved payment, reserved amount will be returned to the user.
+If you cancel a reserved payment, the reserved amount will be returned to the user.
 
-## Cancel Payment By Idempotency Key
+## Cancel payment by idempotency key
 
-You can also cancel payment by sending us the same idempotency key that you've used in the create payment request.
-Use this endpoint when the outcome of create payment request is unknown (e.g. network/server error occured and you didn't get a response). It allows to cancel only those payments that are in `initiated` state. For all other scenarios, please use the [endpoint above](#cancel-payment).
+You can also cancel a payment by sending us the same idempotency key that you've used in the create payment request.
+Use this endpoint when the outcome of the payment creation request is unknown (e.g. network/server error occurred and you didn't get a response). It allows canceling only those payments that are in an `initiated` state. For all other scenarios, please use the [endpoint above](#cancel-payment).
```

Comment:
I think we used British English version elsewhere - "cance**ll**ing"

Reply:
yes, fixed